src/zif_abap_serverless_v1.intf.abap

```diff
@@ -15,9 +15,11 @@ INTERFACE zif_abap_serverless_v1 PUBLIC.
   METHODS
     run
       IMPORTING
-        is_request TYPE ty_http
+        iv_method      TYPE string
+        iv_request_uri TYPE string
```

Comment:
Better, but most consumers will already feature a parsed uri (`path` and `query`). Are you so reluctant to use that?

Reply:
in OpenFaaS is it parsed as part of the protocol specification?

Reply:
In express (which I use as http-framework in the OpenFaaS node image) it is